shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/datasource/pool/metadata/DataSourceJdbcUrlMetaData.java

```diff
@@ -35,6 +35,22 @@
      */
     String getJdbcUrl(T targetDataSource);
     
+    /**
+     * Get Username.
```

Comment:
`Get Username` should rewrite to `Get username`, please care about case sensitive